### PR TITLE
Cohort download tweaks

### PR DESCRIPTION
**Pull request by harimohanraj89**
**Story card:** https://www.pivotaltracker.com/story/show/173188739

## Because

Some missing and clunky bits of info in our district cohort report downloads were making analysis difficult

## This addresses

Tweaks the district cohort report downloads to:
* Add facility type as a new column
* Remove the % signs from values, and add them to the headers instead.

### CODE REVIEW

File: app/views/analytics/districts/show.csv.erb [hunk 1]
```diff
@@ -13,21 +13,21 @@
     'Patients controlled',
     'Patients uncontrolled',
     "Patients didn't attend",
-    'Control rate',
-    'Uncontrolled rate',
-    'Default rate',
+    '% BP Control %',
+    '% BP Uncontrolled',
+    '% Defaulted %',
```
Comment (prabhanshuguptagit) [inline app/views/analytics/districts/show.csv.erb:18]: `%` typo 👾

File: app/views/analytics/districts/show.csv.erb [hunk 2]
```diff
@@ -88,6 +89,7 @@ end).to_csv -%>
     <%=
```
Comment (prabhanshuguptagit) [inline app/views/analytics/districts/show.csv.erb:89]: image

The `All` row is missing a `nil` value under facility type.

File: app/views/analytics/districts/show.csv.erb [hunk 3]
```diff
@@ -65,6 +65,7 @@ end).to_csv -%>
 <%
   headers = [
     "Facility name",
+    "Facility type",
```
Comment (prabhanshuguptagit) [inline app/views/analytics/districts/show.csv.erb:68]: nitpick of the day: `Facility Type`?

File: app/controllers/analytics/districts_controller.rb [hunk 4]
```diff
@@ -84,9 +84,21 @@ def set_dashboard_analytics(period, prev_periods)
   end
 
   def set_facility_keys
-    facilities = @organization_district.facilities.order(:name).pluck(:id, :name).to_h
-
-    @facility_keys = { total: "Total" }.merge(facilities).with_indifferent_access
+    district = {
+      id: :total,
+      name: "Total",
+      type: "District"
```
Comment (prabhanshuguptagit) [inline app/controllers/analytics/districts_controller.rb:90]: image

The `Facility Type` : `District` column feels slightly confusing, can we maybe say **`Total - District`** and not have that column in the totals?

File: app/views/analytics/districts/show.csv.erb (hunk 3 above)
Comment (harimohanraj89) [reply app/views/analytics/districts/show.csv.erb:68]: In fact, changed the other one to `Facility type` for consistency instead.

File: app/controllers/analytics/districts_controller.rb (hunk 4 above)
Comment (harimohanraj89) [reply app/controllers/analytics/districts_controller.rb:90]: I'll just stick a `nil` in there. The fact that it says `<District> Cohort Report` right above makes it clear what the total is 👍

**Comment by harimohanraj89** [comment]
Hey @prabhanshuguptagit, thanks for the feedback. Addressed in 241dbef. Here's what the download looks like now:

Screenshot 2020-06-08 at 2 55 57 PM

File: app/controllers/analytics/districts_controller.rb (hunk 4 above)
Comment (harimohanraj89) [reply app/controllers/analytics/districts_controller.rb:90]: There was also a bug where the facilities' types weren't being printed in there. Fixed now.

File: app/views/analytics/districts/show.csv.erb [hunk 5]
```diff
@@ -13,21 +13,21 @@
     'Patients controlled',
     'Patients uncontrolled',
     "Patients didn't attend",
-    'Control rate',
-    'Uncontrolled rate',
-    'Default rate',
+    '% BP Control %',
```
[the remark this answers is not in the document]
Comment (harimohanraj89) [reply app/views/analytics/districts/show.csv.erb:16]: I have forgotten how to read 🤦

File: app/controllers/analytics/districts_controller.rb (hunk 4 above)
Comment (prabhanshuguptagit) [reply app/controllers/analytics/districts_controller.rb]: Ah, ok. That's what the empty cells were about. :shipit:

**Pull request merged**